packages/valory/skills/dynamic_nft_abci/behaviours.py

```diff
@@ -622,7 +624,7 @@ def async_act(self) -> Generator:
 
         self.set_done()
 
-    def update_layers(self):
+    def update_layers(self):  # pragma: no cover
```

Comment:
The behaviour test for this actually exists, but the call has been commented out in the behaviour. Anyway, since we are not using the IPFS functionality until we have the whitelist server, it is not worthy to worry about this.